test-charts/cron-job/with-s3-volume/values.yaml

```diff
@@ -0,0 +1,10 @@
+bandstand-cron-job:
+  owner: ktech-developer-platform
+  env: test
+  business: ktech
+
+  s3Volume:
+    enabled: true
+    bucketName: bucket-name
+    prefix: bucket/prefix/path
+    authenticationSource: pod
```

Comment:
Can we tweak this to make it more flexible? Instead of a single mount could we do something like:

```
s3Volumes:
  - bucketName: someBucket
     prefix: some/prefix # optional
     mountPath: /mnt/bucket1 # maybe optional
  - bucketName: someOtherBucket
     prefix: some/prefix # optional
     mountPath: /mnt/bucket2 # maybe optional
```